snapshots/PositionManager.Operations.json

```diff
@@ -0,0 +1,25 @@
+{
+  "AllowancePositionManager: approveWithdraw": "46858",
```

Comment:
think this json is big enough now that it warrants allowance posm & position config posm have different jsons w different namespace

Reply:
883990bee184a5ee64095195dd87e65d0bfe72c3